sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/AppBar.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.sample.compose.ui.screens
+
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import co.nimblehq.sample.compose.R
+
+@Composable
+fun AppBar() {
```

Comment:
Shall we create a parameter for `title`? Currently, we use the sample title on both `Home` and `Second` screen

Reply:
Conclusion: We will use "Home" as the title for the Home screen and "Second" as the title for the Second screen